### PR TITLE
fix: remove hard-coded stroke property from mobile icon

### PR DESCRIPTION
**Pull request by cynthiachan**
Removed stroke property from the MobileIcon component because there was a hard coded color in the icon and re-generated icons.

### CODE REVIEW

**Comment by pducolin** [comment]
I would also move these icons somewhere else, they don't fit graphically with other icons.

**Comment by oclero** [comment]
I agree. I like to store eveything related to app branding in the same folder (icons, logo, etc.), so we keep the icon library standardized.

File: src/atoms/icons/index.ts [hunk 1]
```diff
@@ -1,8 +1,8 @@
 /*-- EXPORT ICON COMPONENTS --*/
 export { WindowsIcon } from './WindowsIcon';
 export { WebIcon } from './WebIcon';
-export { UserCircleIcon } from './UserCircleIcon';
 export { WarningIcon } from './WarningIcon';
+export { UserCircleIcon } from './UserCircleIcon';
 export { VaultIcon } from './VaultIcon';
```
Comment (akamkaro) [inline src/atoms/icons/index.ts:6]: order alphabetically please

Comment (oclero) [reply src/atoms/icons/index.ts:6]: Yes I second Matthieu on that: please keep the alphabetical order, so its easier to find the icon in the file.

Comment (cynthiachan) [reply src/atoms/icons/index.ts]: Added.

**Pull request merged**